.github/workflows/checker-framework.yml

```diff
@@ -7,19 +7,19 @@ on:
   pull_request:
 
 jobs:
-  checker-framework:
+  analyse:
```

Comment:
We can rename to just be `check` to save two more letters for phone-friendly CI diagnostics :)

Reply:
done